core/tracker/tracker2/tracker.go

```diff
@@ -166,6 +166,10 @@ const (
 	// This indicates a bug in charon as it is unexpected.
 	msgFetcherSyncContributionFailedSigAggPrepare = "couldn't fetch sync contribution due to no aggregated sync contribution selection, this is unexpected"
 
+	// msgFetcherUnknownError indicates duty failed in fetcher step with some unknown error.
+	// This indicates a bug in charon as it is unexpected.
+	msgFetcherUnknownError = "couldn't fetch due to unknown error"
```

Comment:
can improve this to check for eth2 errors

Reply:
done